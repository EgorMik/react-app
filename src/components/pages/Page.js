import React from 'react';

const Page = () => {
  return (
    <div className={"page"}>
      <p>TODO</p>
      </div>
  );
};

export default Page;