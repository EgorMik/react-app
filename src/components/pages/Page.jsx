import React from 'react';


const Card = () => {
  return (
    <div className={"page"}> </div>
  );
};

export default Card;