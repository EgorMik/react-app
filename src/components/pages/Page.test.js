import React from "react";
import renderer from "react-test-renderer";
import Page from "./Page";

describe("Page", () => {
  it("should renders correctly", () => {
    const tree = renderer.create(<Page name="test" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});