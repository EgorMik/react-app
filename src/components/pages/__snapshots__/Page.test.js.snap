// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page should renders correctly 1`] = `
<div
  className="page"
>
  <p>
    TODO
  </p>
</div>
`;
